src/main/java/edu/ksu/canvas/requestOptions/CreateSisImportOptions.java

```diff
@@ -0,0 +1,38 @@
+package edu.ksu.canvas.requestOptions;
+
+public class CreateSisImportOptions extends BaseOptions {
+
+    public enum ImportType {
+        instructure_csv;
+        // With a standard Canvas install, this option can only be 'instructure_csv', and if unprovided, will be assumed to be so.
+
+        @Override
+        public String toString() { return name().toLowerCase(); }
+
+    }
+
+    public static final String ATTACHMENT = "attachment";
+
+    private final String accountId;
+    private final String filePath;
+    private final ImportType importType;
+    
+    public CreateSisImportOptions(String accountId, String filePath, ImportType importType) {
```

Comment:
It would be helpful if there was an option to supply an `InputStream` or something similar. That way the calling code doesn't have to write a temp file out to disk (although it may wish to anyway).

Reply:
Do you mean an additional parameter (which might or not be used), or replacing the filePath param altogether? The code where this will be used is reading from a file, which then comes handy for the FileBody Multipart bit on the SimpleRestClient. The alternative would be using the raw binary body way. Maybe I should create different methods for both ways?

Reply:
Yeah, I think there's value in both being able to supply a file, but also being able to hand it an `InputStream`